src/osd/scrubber/scrub_backend.cc

```diff
@@ -867,12 +872,11 @@ std::optional<std::string> ScrubBackend::compare_obj_in_maps(
   auto [auths, objerrs] =
     match_in_shards(ho, auth_res, object_error, errstream);
 
-  auto opt_ers =
-    for_empty_auth_list(std::move(auths),
-                        std::move(objerrs),
-                        auth,
-                        ho,
-                        errstream);
+  auto opt_ers = for_empty_auth_list(std::move(auths),
+                                     std::move(objerrs),
+                                     auth,
+                                     ho,
+                                     errstream);
```

Comment:
Unnecessary change.

Reply:
The formatter forced 80-columns limit on the touched files. Will move all those changes to
a specific PR.
And - BTW - I wish we reconsider and allow 85 or 90 char lines. Too many line breaks this way.

Reply:
There's a great deal of intrinsic value in the whole project using the same formatting norms.  I don't think that changing to 85 or 90 character lines is at all worth breaking that property.

Reply:
I agree. I meant allowing that for the whole project. I'd suggest that modern monitors are wide enough to hold multiple 90-cols windows.

Reply:
That's a discussion for ceph-devel then, but fwiw, I really prefer 80 chars because it allows me to fit more files on a single monitor horizontally.